app/main.py

```diff
@@ -1,6 +1,26 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    filtering = os.getenv("FILTERING")
```

Comment:
The checklist recommends using uppercase for constants' names. Consider defining `FILTERING` as a constant in uppercase if it is a fixed value.